src/NzbDrone.Core/Notifications/Telegram/TelegramProxy.cs

```diff
@@ -35,13 +35,13 @@ public TelegramProxy(IHttpClient httpClient, ILocalizationService localizationSe
         public void SendNotification(string title, string message, TelegramSettings settings)
         {
             // Format text to add the title before and bold using markdown
-            var text = $"<b>{HttpUtility.HtmlEncode(title)}</b>\n{HttpUtility.HtmlEncode(message)}";
+            var text = $"*{HttpUtility.HtmlEncode(title)}*\n{HttpUtility.HtmlEncode(message)}";
 
             var requestBuilder = new HttpRequestBuilder(URL).Resource("bot{token}/sendmessage").Post();
 
             var request = requestBuilder.SetSegment("token", settings.BotToken)
                                         .AddFormParameter("chat_id", settings.ChatId)
-                                        .AddFormParameter("parse_mode", "HTML")
+                                        .AddFormParameter("parse_mode", "MarkdownV2")
```

Comment:
I changed the parser to `MarkdownV2` to have the link render properly, as I think the HTML `hrefs` are being escaped in `HTML`-mode when just added to the message. However, this breaks the `SeriesDeleteMessage` as it contains a reserved character `-`. 

Not quite sure how to handle this.

Reply:
Which part of the message? A dash could also be in the series. We could escape the `-`, possibly with just `\-`, but what about `*` and other special characters in markdown? `_` could cause issues, especially if there are two in the title.

Reply:
Still need to determine the impact of this

Reply:
Yes, apologies for the lack of activity here. 

I'll try to mess around with reverting back to HTML. 
Been unable to find a clean solution. 

The problem I had initially with the HTML parser was that the markup would render as raw strings in the message, instead of formatting the message.

Reply:
This is because `HttpUtility.HtmlEncode` is being called on the body text. The message body with the link can't be passed into `SendNotification`, the link needs to be passed in separately and then built in the proxy, which is better because it hides that implementation detail from `Telegram.cs`